pommerman/network/ion_server/network.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+"""IonServer Network handler - This contains functions responsible Server-Client communication (You shouldn't use this file directly)"""
+
+# Imports
```

Comment:
remove. we know what this section is.

Reply:
I dunno, I'll either have to remove them all or none of them. I like some basic classification of the code. It looks more readable to me.